data/compatible_products.yml

```diff
@@ -11,6 +11,104 @@
+          download_url: 'https://managefish.s3.amazonaws.com/managefish-6.1.0.tar.gz?AWSAccessKeyId=AKIAJNYMDLCT3Q3TBS4A&Expires=1625205438&Signature=7FaIdsXsBhXHlRVR91RxQmNC2j4%3D'
+    - title: Jakarta EE 9.1 Web Profile
+      items:
+      - name: Eclipse GlassFish
+        vendor: Eclipse Foundation
+        image: '/images/compatible_products/glassfish_logo.svg'
+        image_width: 195
+        download: 'https://glassfish.org/download'
+        versions:
+        - version: 6.1 RC1
+          compatibility: 'https://glassfish.org/certifications/jakarta-web-profile/9.1/TCK-Results-6.1-RC1'
+          download_url: 'https://download.eclipse.org/ee4j/glassfish/glassfish-6.0.0.zip'
+        - version: 6.0
+          compatibility: 'https://glassfish.org/certifications/jakarta-web-profile/9.1/TCK-Results-6.0'
+          download_url: 'https://download.eclipse.org/ee4j/glassfish/glassfish-6.0.0.zip'
+      - name: Open Liberty
+        vendor: IBM Corporation
+        image: '/images/compatible_products/ibm_open_liberty.svg'
+        image_width: 195
+        download: 'https://openliberty.io/downloads/#runtime_betas'
+        versions:
+        - version: 21.0.0.3-beta, Java 11
+          compatibility: 'https://openliberty.io/certifications/jakartaee/webprofile/9.1/21.0.0.3-beta-Java11-TCKResults.html'
+          download_url: 'https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/tck/2021-02-09_1100/openliberty-jakartaee9-21.0.0.3-beta.zip'
+        - version: 21.0.0.3-beta, Java 8
+          compatibility: 'https://openliberty.io/certifications/jakartaee/webprofile/9.1/21.0.0.3-beta-Java8-TCKResults.html'
+          download_url: 'https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/tck/2021-02-09_1100/openliberty-jakartaee9-21.0.0.3-beta.zip'
+      - name: WildFly
+        vendor: Red Hat
+        image: '/images/compatible_products/red-hat-wildfly-logo.png'
+        image_width: 100
+        download: 'https://wildfly.org/downloads'
+        versions:
+        - version: Preview 23.0.2.Final, Java 11
+          compatibility: 'https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-web-profile.adoc'
+          download_url: 'https://download.jboss.org/wildfly/23.0.2.Final/wildfly-preview-23.0.2.Final.zip'
+        - version: Preview 23.0.2.Final, Java 8
+          compatibility: 'https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-web-profile-jdk8.adoc'
+          download_url: 'https://download.jboss.org/wildfly/23.0.2.Final/wildfly-preview-23.0.2.Final.zip'
+      - name: Apache TomEE
+        vendor: Apache Software Foundation
+        image: '/images/compatible_products/apache_tomee-logo.png'
+        image_width: 195
+        download: 'https://repository.apache.org/content/repositories/orgapachetomee-1184/org/apache/tomee/apache-tomee/9.0.0-M7/apache-tomee-9.0.0-M7-plume.zip'
+        versions:
+        - version: 6.1.0
```

Comment:
The version number is not correct - it should be 9.0.0-M7.

Reply:
Done!